internal/api/v1/service/show.go

```diff
@@ -49,9 +52,32 @@ func Show(c *gin.Context) apierror.APIErrors {
 		return apierror.InternalError(err)
 	}
 
+	serviceConfigurations, err := configurations.ForService(ctx, cluster, service)
+	if err != nil {
+		return apierror.InternalError(err)
+	}
+
+	if len(serviceConfigurations) > 0 {
```

Comment:
hard to grok these bits; can you please explain what are we doing here?

Reply:
I guess you are looking at the inner for loop with the `j` increments ?
Will add code comments.

Reply:
Done with commit b82a5701e8caed448d3e43df07cbe2e35943e01c